tldw_Server_API/app/core/MCP_unified/modules/registry.py

```
@@ -346,16 +373,26 @@ async def check_all_health(self) -> dict[str, ModuleHealth]:
+            if registration.module_instance is not None
+        }
+        tasks = {
+            module_id: registration.module_instance.health_check()
+            for module_id, registration in registrations.items()
+            if registration.module_instance is not None
+        }
 
         if tasks:
             results = await asyncio.gather(*tasks.values(), return_exceptions=True)
 
             for module_id, result in zip(tasks.keys(), results):
```

Ruff (0.15.9)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)